src/main/java/emissary/Emissary.java

```diff
@@ -101,6 +102,15 @@ protected void execute(String[] args) {
         }
         try {
             jc.parse(args);
+        } catch (MissingCommandException e) {
```

Comment:
@ukengineer925 
This was an idea on how to handle the additional verification that Jcommander now does.
Sorry to step on your work. If you have a better idea on how to handle this I am open to it.